example/lib/pages/marker_anchor.dart

```diff
@@ -67,6 +67,9 @@ class MarkerAnchorPageState extends State<MarkerAnchorPage> {
       ),
     ];
 
+    double screenWidth = MediaQuery.of(context).size.width;
+    double minWidth = (screenWidth - 16) / 4;
```

Comment:
Have you tried replacing the `Row` widget with a [Wrap](https://api.flutter.dev/flutter/widgets/Wrap-class.html)?

Reply:
![image](https://user-images.githubusercontent.com/12020398/67010278-4cbdbe00-f128-11e9-9272-d9139d955e0f.png)

Without minWidth, Wrap container will generate this layout. Well we might can just move "Bottom" to 2nd row. Or just keep on with current fix?

Reply:
@johnpryan Oh and I'm very sorry about duplicated PR. #427 already tried on this issue. Will close this PR as soon as confirmed